src/libs/Log.ts

```diff
@@ -82,4 +83,15 @@ const Log = new Logger({
 });
 timeout = setTimeout(() => Log.info('Flushing logs older than 10 minutes', true, {}, true), 10 * 60 * 1000);
 
+AppLogs.configureAppGroupName('group.com.expensify.new');
+AppLogs.registerHandler({
+    filter: '[NotificationService]',
```

Comment:
```suggestion
    filter: 'NotificationService',
```

Reply:
As discussed above let's stick to current `filter` and just slightly modify js logic. I'll mark it as resolved 👍